dhall/src/Dhall/Parser.hs

```diff
@@ -52,7 +56,7 @@ exprA = completeExpression
 data ParseError = ParseError {
       unwrap :: Text.Megaparsec.ParseErrorBundle Text Void
     , input  :: Text
-    }
+    } deriving (Generic, NFData)
```

Comment:
I believe these instances aren't needed anymore, when we error out on parse errors?!

Reply:
Good point!